oskarstark/no-floc-listener/0.1/config/packages/oskarstark_symfony_no_floc_listener.yaml

```diff
@@ -0,0 +1,7 @@
+services:
+    _defaults:
+        autowire: true
```

Comment:
if there is only one service, this can be added to the service itself instead of `_defaults`

Reply:
I think I can remove it entirely, because it has no class members